features/step_definitions/chef-databag.rb

```diff
@@ -5,5 +5,5 @@
 end
 
 Given(/^I create an empty data bag '(.+)'$/) do |databag|
-  run_command_and_stop "knife data bag create #{databag} -z -c config.rb", { fail_on_error: true }
+  run_command_and_stop "knife data bag create #{databag} -z -c config.rb", { fail_on_error: false }
```

Comment:
Why did we need to set this to false? Is this failing with some error that needs to be addressed?

Reply:
we were already passing false to this function if you see, I just added the key because of the  keyword argument and positional argument update in ruby 3.0 and 3.1.
So in the previous ruby versions it was working fine but after 3.0 its throwing error as it expects the key if we pass second argument.